tests/karma/unit/enketo/phone-widget.js

```diff
@@ -0,0 +1,116 @@
+describe('phone-widget', function() {
```

Comment:
So @alxndrsn is the enketo formatting standard to use 4 spaces?  I'll add a comment to the module.